ui/apps/platform/src/Containers/Vulnerabilities/WorkloadCves/components/ExceptionRequestModal/ExceptionRequestForm.tsx

```diff
@@ -215,7 +220,7 @@ function DeferralForm({ cves, scopeContext, onSubmit, onCancel }: DeferralFormPr
                                                     onChange={(_, value) =>
                                                         setExpiry({
                                                             type: 'CUSTOM_DATE',
-                                                            date: value,
+                                                            date: new Date(value).toISOString(),
```

Comment:
Oops - without this format the 'MM/DD/YYYY' string gets sent to the server, which results in an error.